media-ranker/spec/controllers/books_controller_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe BooksController, type: :controller do
+
+  before(:each) do
+    @book = Book.create(title: "Harry Potter and the Sorcer's Stone", author: "J.K.Rolling", description: "The story of the boy who lived")
```

Comment:
You may want to test something like `expect(Book.count).to eq(1)` to make sure it got stored.

And be sure to test that `:create` fails with invalid params too!